pycode/memilio-simulation/memilio/simulation/abm.py

```diff
@@ -22,4 +22,4 @@
 Python bindings for MEmilio ABM model.
 """
 
-from memilio._simulation_abm import *
+from memilio.simulation._simulation_abm import *
```

Comment:
isn't there a conflict or duplication between the imports in abm.py, oseir.py, etc. and the ones in __get_attr__ in __init__.py? seems to me they do the same thing (I guess the __get_attr__ version has priority?)

Reply:
Both have a different use case and thus are not conflicting. When importing python only looks at the file structure and searches the exact file. **get_attr** is for the following case:
`import memilio.simulation as mio
mio.abm.Model()`
In this example **abm** is attribute of memilio.simulation and not searched by looking at the file system. To make that usable we need the **get_attr** function. But it should return the python modules and not the protected modules, because of possible pure python extensions

Reply:
I see. If I understand correctly, your example isn't supported by normal python and has nothing to do with the c++ bindings themselves, right? 

Is it common in python projects to add __get_attr__ to make this work? And if so, shouldn't we add __get_attr__ at the level of memilio as well? It might be confusing if this works:
```
import memilio.simulation
memilio.simulation.abm.Model()
```
but this doesn't:
```
import memilio
memilio.simulation.abm.Model()
```

Reply:
It depends on the project, eg. NumPy follows this convention. This is a decision we have to make. Implementing it at the level of memilio seems harder, because all of our submodules (simulation, epidata,..) have different __init__.py files (that do the same thing) and only one of those files is installed into the environment. 

Yes, this behavior could be confusing. On the other hand, memilio is more a collection of different packages inside a single namespaces, that work mostly indipendently. The submodules of simulation are more likely to be used together. 

Implementing it on level memilio does not seem possible, but i still find it usefull to have it for simulation

Reply:
makes sense. We'll leave it as it is. 

It probably would work on memilio level, but each package would have to be listed in the `__init.py__` of each package, which is a hassle to maintain.